.github/ISSUE_TEMPLATE/00-project-onboarding-checklist-template.md

```diff
@@ -9,22 +9,22 @@ assignees: ''
 
 ## Project Onboarding Checklist - ${Project Name}
 
-<!--Copied from ../../PROJECT_PROGRESSION.md on 2019-08-23 -->
+<!--Copied from ../../PROJECT_PROGRESSION.md on 2019-10-31 -->
 
 **Stage: ${Project Stage}
 
 - [ ] Adopt the OpenJS Foundation Code of Conduct
 - [ ] Update project CoC reporting methods to include OpenJS Foundation escalation path
-- [ ] Transfer official domains to OpenJS Foundation
+- [ ] List official domains that the project commits to transfer to the OpenJS Foundation following graduation
 - [ ] Identify and document other core project infrastructure
-- [ ] If choosing to use a Contributor License Agreement (CLA) or Developer Certificate of Origin (DCO), make selection and implement appropriate tool
+- [ ] If choosing to use a Contributor License Agreement (CLA) or Developer Certificate of Origin (DCO), make selection
 - [ ] Add or Update Governance.md document (required for Impact stage)
 - [ ] Confirm required files in place (CODE_OF_CONDUCT.md, LICENSE.md)
 - [ ] Project Charter is published on website or github
```

Comment:
- [ ] Add [SPDX short IDs to files](https://spdx.org/ids-how) (optional, but helpful)

Reply:
To _all_ files?

Reply:
hopefully not; a SPDX declaration in `package.json` should implicitly cover every file in that package, just like a LICENSE file (that github can detect) should implicitly cover every file the repo.

Reply:
It's not a requirement, but adding it to each file is a [best practice based upon the Kernel](https://www.kernel.org/doc/html/v4.18/process/license-rules.html).  Think of the guidance this way: You don't need to do it (note the "optional") but say you were modifying a file anyhow and saw it wasn't there, it would be a good idea to add it.

The idea of short-form identifiers is that they maintain the license notice even if the file is cherry-picked out of the project.

Reply:
I feel like we should more opinionated about this and go for minimal legal requirements everywhere. This just adds friction and noise otherwise.

Reply:
That's a best practice for the linux kernel, which isn't what i'd call an exemplar for any part of the JS ecosystem (linux is also changed with .patch files, which isn't how much else operates). I very much think we should strive for not just minimal requirements, but minimal recommendations.

I think it is a *bad* idea to add it, because it's extra redundant noise; in practice, files don't exist in isolation from npm packages or from git repos.

Reply:
> I think it is a bad idea to add it, because it's extra redundant noise; in practice, files don't exist in isolation from npm packages or from git repos.

Right. This is I think the point I was trying to make. Can we make OpenJSF as dev-friendly as possible and only recommend the minimal necessary legal requirements. The goal should be to remove everything that isn't 100% necessary.

Reply:
Thank you for both of your comments.

I'd like to suggest we take a step back and recenter on the original comment.  The intent of this change was to document something which already exists in the OpenJS Foundation IP policy.  However, it doesn't necessarily need to be in this particular list, so if it will add more confusion than clarity, I'd have no objections to removing the item from the checklist.

Reply:
Yes. Agreed.

I feel like the reason we're having this conversation here is that there was no room (afaik) for discussing the concrete implementation and day-to-day operations impact of the IP policy. Maybe that's something worth discussing next week at the summit?

Reply:
Suggested a session on these IP-related issues at the collab summit next week [here](https://docs.google.com/document/d/1hdyWxXjCXUe1iEUqaQE8jvt38N6ckQomxTCtgkdisnU/edit). Additional subtopics welcomed.

Reply:
Seems we don't have consensus around this guidance suggestion and have agreement not to include it here for the time being. Marking as resolved.